service/lint.ml

```diff
@@ -0,0 +1,19 @@
+open Current.Syntax
+module Docker = Current_docker.Default
```

Comment:
This will need updating to master (we have multiple Docker engines now). It's easiest just to take the base image as a string.

Reply:
Not entirely sure what you mean by 'take the base image as a string'. Pretty-print it and use that as the host?

Reply:
Ah, actually this isn't a problem. Although we build on multiple engines, we only lint using `Default`.